theme/_provisional.scss

```diff
@@ -2,4 +2,8 @@
 	&.bg-cover, .bg-cover {
 		background-size:cover;
 	}
+
+	.provisional-img-example {
+		width: 100%;
```

Comment:
I don't think that CSS is needed. There is a generic CSS class name called `full-width` or `img-responsive`. Please remove and update the working example